src/main/scala/com.gu.gibbons/UnverifiedUser.scala

```diff
@@ -0,0 +1,43 @@
+package com.gu.gibbons
+
+// ------------------------------------------------------------------------
+import cats.Monad
+import config.Settings
+import java.time.OffsetDateTime
+import model._
+import services._
+// ------------------------------------------------------------------------
+
+/** Deletes keys for users who have been sent a verification email but failed to
+ * verify within 24 hours
+ *
+ * @param email The email service interpreter
+ * @param bonobo The bonobo service interpreter
+ * @param logger The logging service interpreter
+ */
+class UnverifiedUser[F[_]: Monad](
+    settings: Settings,
+    email: EmailService[F],
+    bonobo: BonoboService[F],
+    override val logger: LoggingService[F]
+    ) extends Script[F] {
+
+  import cats.syntax.functor._
+  import cats.syntax.flatMap._
+  import cats.instances.vector._
+  import cats.syntax.traverse._
+
```

Comment:
Seems like kind of a shame to have to include these imports each time. We could add them to `Script[_]` like this (and therefore removing the imports there too):

```scala
abstract class Script[F[_]: Monad]
    extends cats.syntax.FlatMapSyntax
    with cats.syntax.FunctorSyntax
    with cats.syntax.TraverseSyntax
    with cats.instances.VectorInstances {
    // ...
}
```

and then these repeated lines wouldn't be neccessary with the implementations of the `Script[_]` trait.

Reply:
If we're not making this change now, can we make sure it gets caught on a health card to apply later, if it's not already?

Reply:
I think Paul has left several of these comments which are really useful for refactoring so I'll add a refactor card and link to this PR.